presenters/trial.js

```diff
@@ -56,6 +56,10 @@ function decorateDocuments(documents) {
   return documentsByCategory;
 }
 
+function decorateRecords(records) {
+  return _.sortBy(records || [], 'source_id');
```

Comment:
No need to do `records || []` here, as you're already doing it when calling the function in line 84.